examples/arm/executor_runner/CMakeLists.txt

```diff
@@ -173,23 +173,38 @@ execute_process(
   COMMAND
     python "${ET_DIR_PATH}/codegen/tools/gen_oplist.py"
     --model_file_path=${ET_PTE_FILE_PATH}
-    --output_path=${CURRENT_BINARY_DIR}/temp.yaml
+    --output_path=${CMAKE_CURRENT_BINARY_DIR}/temp.yaml
   OUTPUT_VARIABLE CMD_RESULT
 )
-if(NOT CMD_RESULT MATCHES "aten::")
-  set(NO_OPS_IN_FILE "true")
+
+if(CMD_RESULT MATCHES "aten::" OR CMD_RESULT MATCHES "dim_order_ops::")
```

Comment:
As a next step, we are linking both quantized_lib and cortex_m_lib - we can do with only one.